Documentation/dev/libvirt-howto.md

```diff
@@ -163,7 +192,7 @@ alias tectonic="${PWD}/tectonic-dev/installer/tectonic"
 
 Initialize (the environment variables are a convenience):
 ```sh
-tectonic init --config=../libvirt.yaml
+tectonic init --config=libvirt.yaml
```

Comment:
Should we match [the `README`][1] and use the `examples/libvirt.yaml` distributed with the tarball?

[1]: https://github.com/openshift/installer/blame/00862d41c0d5cdb0f92416161651e47ce2481b40/README.md#L36-L44

Reply:
My personal preference is that I like the current instructions to make a new copy of it.  That way local customizations aren't tracked by git.

Reply:
> My personal preference is that I like the current instructions to make a new copy of it.

I don't care either way (I expect users will be able to make this decision on their own).  But however you go here, can you make any `README` updates needed to keep the two consistent?

Reply:
Sure, done!